app/views/decidim/proposals/admin/proposals/_form.html.erb

```diff
@@ -0,0 +1,101 @@
+<div class="card">
+  <div class="card-divider">
+    <h2 class="card-title"><%= title %></h2>
+  </div>
+
+  <div class="card-section">
+    <div class="row column hashtags__container">
+      <%= form.text_field :title, class: "js-hashtags", hashtaggable: true, value: form_presenter.title(extras: false).strip %>
+    </div>
+
+    <div class="row column hashtags__container">
+      <%= form.editor :body, hashtaggable: true, value: form_presenter.body(extras: false).strip %>
+    </div>
+
+    <% if @form.component_automatic_hashtags.any? %>
+      <div class="field">
+        <%= form.label :automatic_hashtags %>
+        <div class="checkboxes hashtags">
+          <% @form.component_automatic_hashtags.each do |hashtag| %>
+            <label>
+              <%= check_box_tag "", "", { checked: true }, { disabled: true } %>#<%= hashtag %>
+            </label>
+          <% end %>
+        </div>
+      </div>
+    <% end %>
+
+    <% if @form.component_suggested_hashtags.any? %>
+      <div class="field">
+        <%= form.label :suggested_hashtags %>
+        <div class="checkboxes hashtags">
+          <%= form.collection_check_boxes :suggested_hashtags, @form.component_suggested_hashtags.map {|hashtag| [hashtag.downcase, "##{hashtag}"]}, :first, :last do |option|
+            option.label { option.check_box(checked: @form.suggested_hashtag_checked?(option.value)) + option.text }
+          end %>
+        </div>
+      </div>
+    <% end %>
+
+    <% if @form.meetings %>
+      <div class="meeting-fields">
+        <div class="row column">
+          <%= form.check_box :created_in_meeting, label: t(".created_in_meeting") %>
+        </div>
+
+        <div class="row column" id="proposal_meeting">
+          <%= form.select :meeting_id,
+                          options_for_select(@form.meetings&.map { |meeting| [present(meeting).title, meeting.id] }, selected: meetings_as_authors_selected ),
+                          { include_blank: true, label: t(".select_a_meeting") },
+                          { multiple: false, class: "chosen-select" } %>
+        </div>
+      </div>
+    <% end %>
+
+    <% if component_settings.geocoding_enabled? %>
+      <div class="row column">
+        <%= form.text_field :address %>
+      </div>
+    <% end %>
+
+    <% if @form.categories&.any? %>
+      <div class="row column">
+        <%= form.categories_select :category_id, @form.categories, prompt: t(".select_a_category") %>
+      </div>
+    <% end %>
+
+    <% if current_participatory_space.has_subscopes? %>
+      <div class="row column">
+        <%= scopes_picker_field form, :scope_id %>
+      </div>
+    <% end %>
+
+    <% if component_settings.attachments_allowed? %>
+      <%= render partial: "decidim/admin/shared/gallery", locals: { form: form } %>
+    <% end %>
+
+    <% if @form.equity_composite_index_percentile %>
```

Comment:
ℹ️ This row is added when the form has a `equity_composite_index_percentile` attribute.